website/docs/test/api.md

```diff
@@ -0,0 +1,31 @@
+| --testCommand <testCommand\>             | Test command (e.g. `maestro test flow.yml`). App performance during execution of this script will be measured over several iterations.        |
+| --bundleId <bundleId\>                   | Bundle id of your app                                                                                                                         |
+| --iterationCount <iterationCount\>       | Amount of iterations to be run. Results will be averaged. (default: 10)                                                                       |
+| --maxRetries <maxRetries\>               | Maximum number of retries allowed over all iterations. (default: 3)                                                                           |
+| --duration <duration\>                   | Duration (in ms) is optional, but helps in getting consistent measures. Measures will be taken for this duration, regardless of test duration |
+| --beforeEachCommand <beforeEachCommand\> | Command to be run before each test iteration                                                                                                  |
+| --afterEachCommand <afterEachCommand\>   | Command to be run after each test iteration                                                                                                   |
+| --beforeAllCommand <beforeAllCommand\>   | Command to be run before all test iterations                                                                                                  |
+| --resultsFilePath <resultsFilePath\>     | Path where the JSON of results will be written                                                                                                |
+| --resultsTitle <resultsTitle\>           | Result title that is displayed at the top of the report                                                                                       |
+| --record                                 | Allows you to record a video of the test. This is useful for debugging purposes.                                                              |
+| --recordBitRate <recordBitRate\>         | Value may be specified as bits or megabits, e.g. '4000000' is equivalent to '4M'.                                                             |
+| --recordSize <recordSize\>               | For best results, use a size supported by the AVC encoder.                                                                                    |
+| --logLevel <logLevel\>                   | Set Log level (choices: "error", "warn", "success", "info", "debug", "trace")                                                                 |
+| -h, --help                               | display help for command                                                                                                                      |
```

Comment:
Is it normal that it displays the end of the line on a new one ?

Reply:
yes 
![Kapture](https://github.com/bamlab/flashlight/assets/15631757/85e99c39-5c26-4c5f-a072-b633ff5f6bda)